src/collection-facets.ts

```diff
@@ -88,9 +99,12 @@ export class CollectionFacets extends LitElement {
     year: false,
   };
 
+  @query('modal-manager') private modalManager!: any;
+
   render() {
     return html`
       <div id="container" class="${this.facetsLoading ? 'loading' : ''}">
+        <modal-manager></modal-manager>
```

Comment:
So basically you want to me to change in this way so that we can add `modal-manager` as `<modal-manager></modal-manager>` in app-root.ts file and get the DOM like `  @query('modal-manager') private modalManager!: any;
` and then pass `$this.modalManager` as a property in `<collection-browser>` component and sub-components?